pkg/apis/batch/types.go

```diff
@@ -347,8 +347,9 @@ type JobSpec struct {
 	// +optional
 	ActiveDeadlineSeconds *int64
 
-	// Optional number of retries before marking this job failed.
-	// Defaults to 6
+	// Specifies the number of retries before marking this job failed.
+	// Defaults to 6, unless backoffLimitPerIndex for  is specified (only Indexed Job).
```

Comment:
```suggestion
	// Defaults to 6, unless backoffLimitPerIndex is specified (only Indexed Job).
```

leftover?

Reply:
Sorry for being extra nitpicky, but couldn't one think that the `Default to` 6 also applies only to indexed jobs? 

What about

```suggestion
	// Defaults to 6, unless backoffLimitPerIndex (only Indexed Job) is specified.
```

Reply:
Applied. Thanks!